docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick start
+
+## What you'll do
+
+- Setup a GitHub repository
+- Activate the GitHub APIs related to self-hosted runner
+- Deploy the [GitHub runner charm](https://charmhub.io/github-runner)
+- Ensure GitHub repository setting are secure
+- Run a simple workflow on the self-hosted runner
+
+## Requirements
+
+- GitHub Account.
+- Juju 3 installed.
+- Juju controller on OpenStack, and a juju model.
```

Comment:
Maybe we could mention somewhere in `index.md` that Openstack is the only type of cloud we currently support/have tested, to avoid frustrating experiences on the user side.
At least for me, deploying to an LXD cloud (I mean using LXD VMs with the `virt-type=virtual-machine` constraint and not containers as we do in the integration tests) did not work out of the box.

Reply:
The problem with LXD cloud is due to it uses containers by default. I am thinking about adding a how-to guide for running on LXD. What do you think?

Reply:
Yes, if it works, of course, feel free to add.